dashboard/test/factories/pd_factories.rb

```diff
@@ -856,7 +856,7 @@
       with_full_form_data
     end
 
-    trait :withdrawn do
+    trait :declined do
```

Comment:
It seems this was called `withdrawn` to match the application status for testing, but it was confusing me so I renamed it to be more descriptive and updated the [test](https://github.com/code-dot-org/code-dot-org/pull/21069/files#diff-e9135aa394456ab3ad7adabce987f3ecL53) to be more explicit.